app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, collection_of_coins, marks],
+    "immutable": [pi, lucky_number, one_is_a_prime_number, name, profile_info]
+}
```

Comment:
The variable `collection_of_coins` is categorized as mutable, but sets in Python are technically mutable in terms of their ability to have items added or removed. However, if the task requires strict immutability (like tuples or strings), this classification might need reconsideration. Ensure this aligns with the task requirements.